src/stan/services/util/run_sampler.hpp

```diff
@@ -62,6 +62,9 @@ namespace stan {
         clock_t end = clock();
         double warm_delta_t = static_cast<double>(end - start) / CLOCKS_PER_SEC;
 
+        writer.write_adapt_finish(sampler);
```

Comment:
Changing the message and the format of the output file will break a bunch of tests.  For now can you add a `write_sampler_state` function that doesn't output "Adaption Finished" and then call that function here?